src/plugins/launchdarkly/index.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from "path";
+
+module.exports = function () {
```

Comment:
If we are using `import` in the file so, let's use export (default probably) here instead of module.export, does it work?

Reply:
Ok fixed